users/views.py

```diff
@@ -53,3 +53,20 @@ def authenticated_test(request):
         return HttpResponse("your number is {}".format(num))
     else:
         return redirect("login")
+
+
+def homepage(request):
+
+    return render(request, "users/homepage.html" )
+
+def friend(request):
+
+    return render(request, "users/homepage.html" )
+
+def post(request):
```

Comment:
I think that blogging should be in a seperate app

Reply:
Homepage has 3 buttons of  options, so I create barbase.html for convenience. I can try to use base.html in homepage.

Reply:
Got it. I'll modify it later.

Reply:
Thanks!